### PR TITLE
use refresh token instead of access token

### PR DESCRIPTION
**Pull request by paddie**
# Description of change
(write a short description or paste a link to JIRA)

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: tap_linkedin_ads/client.py [hunk 1]
```diff
@@ -94,43 +94,41 @@ def raise_for_error(response):
 
 
 class LinkedinClient(object):
-    def __init__(self, access_token, user_agent=None):
-        self.__access_token = access_token
+    def __init__(self, client_id, client_secret, refresh_token, user_agent=None):
```
Comment (JingLin0) [inline tap_linkedin_ads/client.py:97]: why not just delete the user_agent, since we will not use it.

File: tap_linkedin_ads/__init__.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python3
 
+# type: ignore
```
Comment (JingLin0) [inline tap_linkedin_ads/__init__.py:3]: what is stubs?

**Review by przywartya — APPROVED** [review]

**Review by JingLin0 — APPROVED** [review]

File: tap_linkedin_ads/__init__.py (hunk 2 above)
Comment (paddie) [reply tap_linkedin_ads/__init__.py:3]: stubs?

File: tap_linkedin_ads/client.py (hunk 1 above)
Comment (paddie) [reply tap_linkedin_ads/client.py:97]: meh, good point.

File: tap_linkedin_ads/__init__.py (hunk 2 above)
Comment (paddie) [reply tap_linkedin_ads/__init__.py:3]: ahh, the commit name :) . Stubs is something that you can do to wrap some code such that is has typing information. Basically what you do if you are, say, mixing normal javascript and typescript. You will then create a wrapper for the non-typed javascript that is typed, and then call the wrapper from typescript and everyone is happy - although a bit confused.

File: tap_linkedin_ads/client.py (hunk 1 above)
Comment (paddie) [reply tap_linkedin_ads/client.py:97]: hmm.. we are actually passing it in in the runner and adding it as a header below

**Pull request merged**